README.rst

```diff
@@ -31,6 +32,12 @@ Details of the config.yaml file are in `edx-platform/conf/locale/config.yaml
 Changes
 =======
 
+v0.3.3
+------
+
+* ``i18n_tool detect`` command added. This command checks if the GitHub English
```

Comment:
I'm not sure what should be changed in setup.py. The version is set in i18n/__init__.py (0.3.3) is there something else?